### ISSUE TITLE
Segfault after sending request in "Try it"

### ISSUE THREAD

**Issue by aviskase** [stoplightio/studio#364]
**Describe the bug**
When I click "Send" for anything in our API in "Try it" studio crashes with segfault.

**To Reproduce**
Happens even with the smallest version of our API: [api.zip](https://github.com/stoplightio/studio/files/4364649/api.zip). Host is internal deployment (we don't have external facing hosts)
I suspect SSL validation: even though we use HTTPS, for test environments we need to disable SSL validation in clients.

For Petstore example "Try it" works fine.

logs:
```
➜ ./stoplight-studio-linux-x86_64_f5650f59acd0a03c6efb7de930c1f9b8.AppImage 
(electron) 'getName function' is deprecated and will be removed. Please use 'name property' instead.
(electron) The default value of app.allowRendererProcessReuse is deprecated, it is currently "false".  It will change to be "true" in Electron 9.  For more information please check https://github.com/electron/electron/issues/18397
00:28:19.385 › Checking for update
00:28:19.432 › Generated new staging user ID: 55d69007-36a2-51b7-9bb0-3d77f797199b
00:28:20.750 › Update for version 1.10.2 is not available (latest version: 1.10.2, downgrade is allowed).
Segmentation fault (core dumped)
```
`/var/log/syslog/
```
Mar 22 00:28:42 aviskase-xps15 kernel: [24293.586531] stoplight-studi[22637]: segfault at 8 ip 0000563bc7977b4d sp 00007ffdb0437ca0 error 4 in stoplight-studio[563bc7970000+53ce000]
```

**Environment (remove any that are not applicable):**
 - Studio version: 1.10.2
 - OS: Ubuntu 19.10

**Comment by philsturgeon** [comment]
Thanks for this, I've elevated it to the appropriate peoples.

**Comment by vintprox** [comment]
Yeah, I am completely misguided now, when I just happened to find some decent documentation suite for REST APIs, which is Stoplight, and it basically crashes when I send simple request. I am looking forward for the update! 💯 🔆

**Comment by philsturgeon** [comment]
@vintprox hey there, sorry for the delay. We've not heard anything more from anyone else about this issue and I've not been able to recreate it, so.... not sure what to do here. Could you try and recreate it in v1.12?

**Comment by philsturgeon** [comment]
Agh ok no it's happening in Studio Desktop v1.12.0.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/67381/85556988-a9faff00-b627-11ea-80c0-f99d719cde57.png">

**Comment by weleoka** [comment]
Whenever a "Try it" request to an https server with an untrusted self-signed certificate is made it will crash on Stoplight v1.13.0 running as an Appimage on Debian 10 Sid `stoplight-studio-linux-x86_64.AppImage`.

It leaves a "Segmentation Fault" and the process was terminated SIGSEGV. 

Tell me how I can get better logs out of this or if there is a supply of the source code somewhere. 

Also, on a side note, the About section in the App says Copyright 2019.

**Comment by philsturgeon** [comment]
Just to let you folks know, a PR has resolved the segfault, and we’re just figuring out how to handle the scenario properly. Self signed certs are certainly a thing which a lot of people use for dev environments, so we want to make sure we cater to that properly. Updates soon!
